projects/fal/tests/test_apps.py

```diff
@@ -47,6 +48,23 @@ def addition_app(input: Input) -> Output:
     return Output(result=input.lhs + input.rhs)
 
 
+@fal.function(
+    keep_alive=60,
+    machine_type="S",
+    serve=True,
+    max_concurrency=1,
+    requirements=[f"pydantic=={pydantic_version}"],
+    _scheduler="nomad",
+)
+def nomad_addition_app(input: Input) -> Output:
+    print("starting...")
+    for _ in range(input.wait_time):
+        print("sleeping...")
+        time.sleep(1)
+
+    return Output(result=input.lhs + input.rhs)
```

Comment:
Could avoid copying stuff over by using parametrization, but I think this is fine for now as is.